mainapp/templates/mainapp/contributor_form.html

```diff
@@ -1,48 +1,199 @@
+           <tr>
+              <td>
+                <img class="payment-logo" src="https://donation.cmdrf.kerala.gov.in//assets/images/gateway-sbi.jpg"/>
+                <br/>
+                <b>STATE BANK OF INDIA</b>
+              </td>
+              <td>
+                  Internet Banking - 40 banks supported<br/>
+                  Domestic Credit and Debit Cards supported<br/><br/>
+
+                  payment charges: Nil
+              </td>
+              <td>
+                  A/c Number : 67319948232<br/>
+                  A/c Name: Chief Minister’s Distress Relief Fund<br/>
+                  Branch: City Branch, Thiruvananthapuram<br/>
+                  IFSC : SBIN0070028
+              </td>
+              <td>
+                  <img class="qr-code" src="https://donation.cmdrf.kerala.gov.in//assets/images/upi-sbi.png"/>
+                  <br/><br/>
+                  keralacmdrf@sbi
+              </td>
+            </tr>
+
+            <tr>
+                <td>
+                  <img class="payment-logo" src="https://donation.cmdrf.kerala.gov.in//assets/images/gateway-paytm.jpg"/>
+                  <br/>
+                  <b>PAYTM</b>
+                </td>
+                <td>
+                    <b>SMART URL SERVICE</b><br/>
+                    <a href="http://m.p-y.tm/Kerala-CMDRF" target="_blank">m.p-y.tm/Kerala-CMDRF</a><br/><br/>
+                     payment charges: Nil
+                </td>
+                <td>
+                </td>
+                <td>
+                    <img class="qr-code" src="https://donation.cmdrf.kerala.gov.in//assets/images/upi-paytm.png"/>
+                    <br/><br/>
+                    PAYTM app only
+                </td>
+              </tr>
+
+              <tr>
+                 <td>
+                   <img class="payment-logo" src="https://donation.cmdrf.kerala.gov.in//assets/images/gateway-hdfc.jpg"/>
+                   <br/>
+                   <b>HDFC BANK</b>
+                 </td>
+                 <td>
+                     Internet Banking - 53 banks supported<br/>
+                     Domestic Credit and Debit Cards supported<br/>
+                     <strong>INTERNATIONAL</strong> Credit and Debit Cards supported<br/><br/>
+
+                     payment charges: Nil
+                 </td>
+                 <td>
+                     A/c Number : 50100248534439<br/>
+                     A/c Name: Chief Minister’s Distress Relief Fund<br/>
+                     Branch: Vattiyoorkavu, Trivandrum<br/>
+                     IFSC : HDFC0002485</br>
+                     SWIFT CODE: HDFCINBBXXX
+                 </td>
+                 <td>
+                     No QR Code
+                 </td>
+               </tr>
+    </tbody>
+  </table>
+  <br />
+  <h4>Donations in Kind</h4>
+  <p>
+      <img style="max-width:100%" src="https://images-eu.ssl-images-amazon.com/images/G/31/traffic/PC_editorial._CB1198675309_.jpg" alt="Amazon_Cover"/> </br></br>
```

Comment:
`br` comment applicable in multiple places.

Reply:
@bhumilharia thanks for bringing it to notice. can we replace <p></p> with <br/>.

@aravindharinair

Reply:
trying to :D